src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -54,6 +61,23 @@ public void Configure()
             }
         }
 
+        private void AddPackageExecutablePath()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                if (!_firstTimeUseNoticeSentinel.Exists())
```

Comment:
If the user turns off first run, won't you keep doing this for every run?

Reply:
yes, however configure constructor are doing things like this already

Reply:
What do you mean?

Reply:
CLI is checking file's and env's existence multiple 
 time during https://github.com/dotnet/cli/blob/master/src/dotnet/Program.cs#L204 to L219  . If we are not worry about it today, it is probably not the bottom neck of the performance.